README.md

```diff
@@ -60,7 +74,7 @@ You can find older version of Chucker here: [![JitPack](https://jitpack.io/v/Chu
 Don't forget to check the [changelog](CHANGELOG.md) to have a look at all the changes in the latest version of Chucker.
 
 * Compatible with **OkHTTP 4**
-* **API >= 16** compatible
+* **API >= 21** compatible
```

Comment:
Should we add a note that the last API 16+ version is Chucker `3.4.0`?

Reply:
IMHO readme and all of the stuff should be about the current state of the repository at a specific commit. If someone wants to know about `3.4.0` or whichever version they should check the specific commit or a tag.

Reply:
> IMHO readme and all of the stuff should be about the current state of the repository at a specific commit

As a general rule, I agree.

However the Min SDK bump can be a bummer for several users. Offering a previous version as an alternative could be a nice way to don't entice them away (similarly to what happened with the AndroidX migration).

Reply:
Ok, makes sense. Would you see it as a separate section in readme?

We should probably add to this info a period for how long we plan to support `3.x` actively. If I remember correctly it was supposed to be half a year? @vbuberen do you remember? I think you should know better.

Reply:
Yeah even a separate section would work 👌

Reply:
We mentioned bumping minSDK in 3.3.0 release https://github.com/ChuckerTeam/chucker/blob/develop/CHANGELOG.md#version-330-2020-09-30

We can simply mention that Chucker 3.x has minSDK 16 and Chucker 4.x has minSDK 21. It will save us from doing some changes when we need to release something like 3.4.1, for example.